app/controllers/catalog_controller.rb

```diff
@@ -64,7 +64,10 @@ class CatalogController < ApplicationController
     config.add_show_tools_partial(:bookmark, partial: "bookmark_control", if: :render_bookmarks_control?)
     config.add_show_tools_partial(:citation)
 
-    # config.add_nav_action(:bookmark, partial: "blacklight/nav/bookmark", if: :render_bookmarks_control?)
+    config.add_nav_action(:new_search, partial: "shared/nav/new_search")
```

Comment:
@paulhagon I've moved your navbar changes here to follow a more Blacklight-centric way.